fdbserver/Knobs.cpp

```diff
@@ -219,6 +219,8 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( DD_VALIDATE_LOCALITY,                                 true ); if( randomize && BUGGIFY ) DD_VALIDATE_LOCALITY = false;
 	init( DD_CHECK_INVALID_LOCALITY_DELAY,                       60  ); if( randomize && BUGGIFY ) DD_CHECK_INVALID_LOCALITY_DELAY = 1 + deterministicRandom()->random01() * 600;
 	init( DD_ENABLE_VERBOSE_TRACING,                           false ); if( randomize && BUGGIFY ) DD_ENABLE_VERBOSE_TRACING = true;
+	init( DD_TEAMS_INFO_PRINT_INTERVAL,                         1800); // Print detailed DD team status every 30 mins
```

Comment:
BUGGIFY it to small value for simulation test.

Both knobs need BUGGIFY.

Reply:
What's the purpose of this?

Reply:
If you don't BUGGIFY,  the simulated time may never reach DD_TEAMS_INFO_PRINT_INTERVAL. 
The printing code may not trigger as many times as you want.

Basically, without BUGGIFY, your code may not get tested in simulation.

Reply:
Done.